pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -30,9 +31,13 @@ Examples:
 
 	# Starts build from build matching the name "3bd2ug53b"
 	$ osc start-build --from-build=3bd2ug53b
+
+	# Starts build from buildConfig matching the name "3bd2ug53b" and streams its logs
+	$ osc start-build 3bd2ug53b --stream-logs
```

Comment:
Add this behavior to the end to end test so the user can see what is happening with their build in realtime